src/javascript/transpile.ts

```diff
@@ -259,3 +256,12 @@ export function rewriteParams(output: Sourcemap, body: Node, params: Params, inp
     output.replaceLeft(node.start, node.end, JSON.stringify(params[name]));
   }
 }
+
+/**
+ * Annotate a path to a local import or file so it can be reworked server-side.
+ */
+function annotatePath(uri: string): string {
+  return !uri.startsWith(".") || /(?:[.][/]|(?:[.][.][/])+)(_jsr|_node|_npm|_observablehq)[/]/.test(uri)
```

Comment:
This should be `isAssetPath` or whatever that’s `['./', '../', '/']`. Don’t want to match against `.foo`.